core/modules/checkout/components/Payment.ts

```diff
@@ -47,6 +50,9 @@ export const Payment = {
     }
     this.changePaymentMethod()
   },
+  beforeDestroy () {
+    this.$bus.$off('checkout-after-load', this.onCheckoutLoad)
```

Comment:
we would like to get rid of this weird even bus... can you try to do some workaround?